### DISCUSSION TITLE
Superscript/subscript in Pub tiles/headings?

### DISCUSSION THREAD

**Discussion by aimeediehl** [knowledgefutures/pubpub#2914] [Support]
Is there a way to use superscript/subscript characters in Pub titles and descriptions? For example, I'd like to be able to use meters cubed (m3 but with superscript) but can't figure out if I'm missing something or if this capability doesn't exist. Same goes for chemical symbols with subscripts (H2O, CO2).

This might actually be a feature request if these and other special characters can't display right now.

Thanks for your help.

**Comment by kyledenlinger** [comment]
Copying and pasting superscript/subscript text into the title works. For copying:

Superscript: `⁰¹²³⁴⁵⁶⁷⁸⁹`
Subscript: `₀₁₂₃₄₅₆₇₈₉`

If you're on a Mac, you can use character selector. When editing the title, go to Edit > Emoji and Symbols in your browser, then Customize List. Turn on ①Digits, and you'll find your super/subscript characters there (as well as many others you can use).
